src/NUnitFramework/framework/Internal/Execution/TextMessageWriter.cs

```diff
@@ -54,10 +54,14 @@ public class TextMessageWriter : MessageWriter
         /// Length of a message prefix
         /// </summary>
         public static readonly int PrefixLength = Pfx_Expected.Length;
-        
+
         #endregion
 
+        #region Instance Fields
         private int maxLineLength = DEFAULT_LINE_LENGTH;
+        private bool diffTypes = false;
+        private string _expectedType, _actualType;
```

Comment:
If the corresponding properties are eliminated, these instance fields don't seem to be needed either since they are only used locally in various methods as far as I can see.